### ISSUE TITLE
Is the continuous batching function enabled by default in vllm?

### ISSUE THREAD

**Issue by SeibertronSS** [vllm-project/vllm#543]
Is the continuous batching function enabled by default in vllm? Can this feature be turned on or off selectively?

**Comment by zhuohan123** [comment]
Yes, this is enabled by default and cannot be turned off. Turning off continuous batching requires a rewrite of our system architecture, which also brings no benefit in performance. Therefore, we did not implement this.
